modules/distr-tracing-tempo-configuring-tempostack-metrics-and-alerts.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * distr-tracing-tempo-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-tempostack-metrics-and-alerts_{context}"]
+= Configuring Tempostack metrics and alerts
+
+You can enable metrics and alerts of TempoStack instances.
+
+.Prerequisites
+
+* Monitoring for user-defined projects is enabled in the cluster. See xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects].
+
+.Procedure
+
+. To enable metrics of a TempoStack instance, set the `spec.observability.metrics.createServiceMonitors` field to `true`.
++
+.createServiceMonitors
+====
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+spec:
+  observability:
+    metrics:
+      createServiceMonitors: true
+----
+====
+
+. To enable alerts for a TempoStack instance, set the `spec.observability.metrics.createPrometheusRules` field to `true`:
++
+.createPrometheusRules
+====
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: sample
+spec:
+  observability:
+    metrics:
+      createPrometheusRules: true
+----
+====
+
+.Verification
+
+. In the OpenShift web console, go to *Observe* -> *Targets*, filter for *Source: User*, and check that *ServiceMonitors* in the format `tempo-$<tempostack_instance_name>-$<component>` have the *Up* status.
```

Comment:
`tempo-$<tempostack_instance_name>-$<component>` looks a bit odd to me, is this the format we use for other docs, or is there a better way to describe this format? It'll look something like `tempo-myexample-ingester` for example, if the TempoStack is called `myexample`.